backend/api/Services/ReportService.cs

```diff
@@ -50,5 +66,27 @@ public async Task<IEnumerable<Report>> ReadAll()
                 report => report.Id.Equals(id, StringComparison.Ordinal)
             );
         }
+
+        private void OnMissionUpdate(object? sender, MqttReceivedArgs mqttArgs)
+        {
+            var mission = (IsarMission)mqttArgs.Message;
+            _logger.LogInformation(
+                "{time} - Mission {id} updated",
```

Comment:
Is there some parallell to fstring that can be used so that we can write {mission.Timestamp} instad of {time}

Reply:
Yes, but the logger recommends this format, not sure why.
`$"{mission.Timestamp - Mission etc"` would be the equivalent

Reply:
Then we can probably assume that the logger recommends it for a reason and follow that.